<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceManager</class>
 <widget class="QWidget" name="ServiceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cmbServiceNames">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>校园网</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国移动</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国联通</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="leOperatorUserId">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOperatorPwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConfirm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
